node/node.go

```diff
@@ -644,6 +645,39 @@ func (n *Node) OpenDatabase(name string, cache, handles int, namespace string) (
 	return rawdb.NewLevelDBDatabase(n.config.ResolvePath(name), cache, handles, namespace)
 }
 
+// OpenDatabaseWithFreezerRemote opens an existing database with the given name (or
+// creates one if no previous can be found) from within the node's data directory,
+// also attaching a chain freezer to it that moves ancient chain data from the
+// database to immutable append-only files. If the node is an ephemeral one, a
+// memory database is returned.
+func (n *Node) OpenDatabaseWithFreezerRemote(name string, cache, handles int, freezer string) (ethdb.Database, error) {
+	if n.config.DataDir == "" {
+		return rawdb.NewMemoryDatabase(), nil
+	}
+	root := n.config.ResolvePath(name)
+	u, err := url.Parse(freezer)
+	if err != nil {
+		return nil, err
+	}
+	switch u.Scheme {
+	case "http", "https":
+	case "ws", "wss":
+	case "stdio":
+	case "":
+		// IPC
+		switch {
+		case freezer == "":
```

Comment:
I don't know about this. I'm not super comfortable providing this 'hidden' default.

Reply:
in reality you're always checking for ipc filepath and any format  restriction you put on it is hidden and arb. really

Reply:
I suppose it's tricky here 🤔

Reply:
err mis read the code yeah I think we should yank it and fail

Reply:
Done with https://github.com/etclabscore/core-geth/pull/103/commits/5d48b2dcb13526d436f17e16af17df534a14a5bc